Makefile

```diff
@@ -10,13 +10,19 @@ NON_MATCHING ?= 0
 ORIG_COMPILER ?= 0
 # Keep .mdebug section in build
 KEEP_MDEBUG ?= 0
+# Disassembles the full ROM instead of a subset
```

Comment:
```suggestion
# Disassembles the full ROM instead of a only required files
```
Maybe something like that?

Reply:
```suggestion
# Disassembles the full ROM instead of just non-decomped files
```
perhaps? Would prefer something describing what the actual subset is.